sdk/appconfiguration/app-configuration/package.json

```diff
@@ -77,7 +77,7 @@
     "@azure/core-http": "^1.0.0",
     "@azure/core-paging": "^1.1.0",
     "@azure/core-tracing": "1.0.0-preview.7",
-    "@azure/identity": "^1.0.0",
+    "@azure/identity": "^1.1.0-preview1",
```

Comment:
do we want to have the version of a GA package in master depending on a preview version?

Reply:
@schaabs - did we decide how we wanted to do this?

Reply:
Since these are dev-dependencies I think that the preview version should be stable enough to use here. The changes in this preview are generally additive, and so I think the risk would be low to development and tests. However, I would leave this up to each language to independently assess the risk of this.

Reply:
In this case it's been changed in "dependencies". Looks like that might have been a mistake?

Reply:
Looking closer it seems that some client libraries (including this one) incorrectly have taken a non-dev dependency on @azure/identity. For now I wouldn't update these dependencies and I think we should file an issue to remove these dependencies, or make them dev-dependencies.